cup_of_join.py

```diff
@@ -0,0 +1,21 @@
+def join(*list_param, sep="-"):
```

Comment:
1. Please add documantation.
2. Is there a way to simplify the function? (for example, by using list comprehension?)